entries/static/entries/css/dashboard.css

```diff
@@ -0,0 +1,104 @@
+* {
```

Comment:
Forgot to add the CSS for the dashboard, managed to salvage this from `staticfiles`